new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -731,6 +737,162 @@ public void testCreateNewCluster() {
 
   }
 
+  /**
+   * Test changes the WebLogic credentials and verifies the servers can startup and function with changed credentials.
+   * a. Creates new WebLogic credentials using WLST.
+   * b. Creates new Kubernetes secret for WebLogic credentials.
+   * c. Patch the Domain Resource with new credentials, restartVerion and introspectVersion.
+   * d. Verifies the servers in the domain restarted and accessing the admin server console with new password works.
+   * e. Verifies the the admin server console access with old credentials fail.
+   */
+  @Order(3)
+  @Test
+  @DisplayName("Test change WebLogic admin credentials for domain running in persistent volume")
+  public void testCredentialChange() {
+
+    final String domainUid = "mydomain";
+
+    final String adminServerName = "admin-server";
+    final String adminServerPodName = domainUid + "-" + adminServerName;
+
+    final String managedServerNameBase = "ms-";
+    String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+    int replicaCount = 3;
+
+    logger.info("Getting node port for default channel");
```

Comment:
getting t3channel port here, fix the log message

Reply:
fixed